gui/doc/viselements.json

```diff
@@ -899,6 +899,16 @@
             "type": "str",
             "doc": "Allows the styling of table cells.\nSee <a href=\"#styling\">below</a> for details."
           },
+          {
+            "name": "tooltip",
+            "type": "str",
+            "doc": "The name of the function that will calculate the cell tooltips.\nSee <a href=\"#cell%20tooltip\">below</a> for details."
```

Comment:
"that will calculate the cell tooltips" -> "that must return a tooltip text for a cell"